packages/core/package.nls.json

```diff
@@ -91,6 +91,7 @@
     "AWS.command.appBuilder.deploy": "Deploy SAM Application",
     "AWS.command.appBuilder.build": "Build SAM Template",
     "AWS.command.appBuilder.searchLogs": "Search Logs",
+    "AWS.command.appBuilder.tailLogs": "Tail Logs",
```

Comment:
We should probably use the same wording (and actually the same `AWS.foo.bar` ide, instead of multiple) in all places: https://github.com/aws/aws-toolkit-vscode/blob/8d48a3c3441c09bfb574aeb5d680f4114c5e0274/packages/core/package.nls.json#L181-L182

However, not a blocker for this PR. Needs a decision about which wording to use in both places.

Reply:
The difference here is that the one you linked shows "when you have a log group selected" (therefore the wording is "tail log group" as in "tail *this* log group"), while this new one is when you have a *function* selected. Maybe this could be "Tail function logs" instead of just "Tail logs". The inspiration was the other existing string right above "Search Logs" (with its corresponding "Search Log Group"), which is a similar behavior, but that starts a search instead of using live tail.


Let me know if this difference makes sense or if we should still try to unify the wording.

Reply:
> The inspiration was the other existing string right above "Search Logs"

Yup I noticed that too, and I like the brevity of "Logs" instead of "Log Group". For now let's leave it as you have in this PR.